orttraining/orttraining/core/graph/mixed_precision_transformer.cc

```diff
@@ -70,10 +66,33 @@ bool IsFP32(const std::unordered_map<std::string, std::vector<int>>& map, std::s
   }
 }
 
+static const std::string Loss_Scale_Input = "loss_scale";
+static const std::string Scaled_Loss_Node = "scaled_loss";
+static const std::string Scaled_Loss_Grad_Mul_Node = "scaled_loss_Grad/Mul_0";
+static const std::string Scaled_Loss_Grad_ReduceSum_Node = "scaled_loss_Grad/ReduceSum_1";
+
```

Comment:
This is not reliable way to check loss subgraph by checking the names of inputs/outputs. Right?

Reply:
You are right. Maybe I can try to get these node from loss_scale input by calling GetConsumers from graph.